ballerina/jetstream_client.bal

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
There is normal NATS client and other files: https://github.com/ballerina-platform/module-ballerinax-nats/blob/master/ballerina/client.bal
How can I differentiate otherwise.

Reply:
Missed that.